src/app/qgslayerstylingwidget.h

```diff
@@ -171,6 +172,7 @@ class APP_EXPORT QgsLayerStylingWidget : public QWidget, private Ui::QgsLayerSty
     QgsMapLayer *mCurrentLayer = nullptr;
     QgsLabelingWidget *mLabelingWidget = nullptr;
     QgsMeshLabelingWidget *mMeshLabelingWidget = nullptr;
+    QPointer<QgsRasterLabelingWidget> mRasterLabelingWidget;
```

Comment:
Any reason to make this one a `QPointer` while the others are not?

Reply:
It seems cleaner to me. The others should probably be too